README.rst

```diff
@@ -3,20 +3,31 @@ Building and uploading numpy wheels
 ###################################
 
 The wheel builds are currently done by Azure Pipelines. Options that may be
-used for future builds are Travis CI and Appveyor. The progress and success of
-builds may be monitored at the following locations:
+used for future builds are TravisCI and Appveyor. Note that currently TravisCI
+and Appveyor are not triggered, if you want to enable them you will need to do
+so in the settings/Webhooks.
 
-Azure Pipelines at
-https://dev.azure.com/numpy/numpy/_build?definitionId=8&_a=summary&view=runs
+**Build process pages**
 
-Travis CI at
-https://travis-ci.org/MacPython/numpy-wheels
+- Azure Pipelines at
+  https://dev.azure.com/numpy/numpy/_build?definitionId=8&_a=summary&view=runs
 
-The Appveyor at
-https://ci.appveyor.com/project/matthew-brett/numpy-wheels
+- Travis CI at
+  https://travis-ci.org/MacPython/numpy-wheels
 
-The driving github repository is
-https://github.com/MacPython/numpy-wheels
+- The Appveyor at
+  https://ci.appveyor.com/project/matthew-brett/numpy-wheels
+
+- The driving github repository is
+  https://github.com/MacPython/numpy-wheels
+
+**Uploaded file locations**
+
+- Release builds at
+  https://anaconda.org/multibuild-wheels-staging/numpy/files
```

Comment:
Just noticed that the 1.18.4 wheels there still have `.dev0` in the name, so is it also triggered for non-release builds?

Reply:
Yes, those wheels shouldn't be there and need to be removed. I think they have been triggered by merges to the master branch as the azure configuration currently points to the HEAD if the 1.18.x branch. We probably need to configure the pipeline not to upload on merges as that makes it hard to work on the repo. We should also configure things so that master uploads to the nightly directory when triggered directly.

I think @mattip has permissions to delete those files. We should probably get permissions for a few more folks to raise the bus factor.

Reply:
I'll go ahead and point the master branch to master and see if that makes a difference.

Reply:
I also don't know who owns the Azure integration here, I don't see the usual option to remove it.

Reply:
Now it is building master but still uploading to the wrong directory when triggerd, so that needs fixing. The directory should be made to depend on the branch.

Reply:
Do you both have anaconda.org accounts/monikers I can add to admin there?

Reply:
I do have permissions to remove the files, but probably best to give you both access anyway. I'll check back later.

Reply:
I have an account as charlesr.harris.

Reply:
Alright, you have admin there now